src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/SaveAsExcelFileStreamWriterHelper.cs

```diff
@@ -269,6 +270,7 @@ public void Dispose()
 
             private XmlWriter writer;
```

Comment:
Minor: we should again move these private fields to the top of the class definition